configure

```diff
@@ -134,19 +134,16 @@ sub configureRXXR2 {
   else {
     &log("Setting up rxxr2");
 
-    &log("Downloading");
-    &chkcmd("wget http://www.cs.bham.ac.uk/~hxt/research/rxxr2/rxxr2.zip -O rxxr2.zip");
-    &chkcmd("unzip rxxr2.zip"); # Unzips to 'rxxr/'
-
+    &log("Cloning repo");
+    &chkcmd("git clone https://github.com/ConradIrwin/rxxr2");
```

Comment:
This makes me a little nervous without pinning it to a specific release or version. Can you add a follow-up `checkout` to pin it?

Reply:
It would also be good to explicitly name the "clone-to" directory (`git clone X local-name`), and might as well make that a variable so the subsequent `cd` and `rm` are more stable.

Reply:
> This makes me a little nervous without pinning it to a specific release or version. Can you add a follow-up `checkout` to pin it?

I don't quite understand this part

Reply:
> I don't quite understand this part

The current command will check out whatever the most recent commit is to the main branch of the rxxr2 repository. In theory, they could introduce incompatible changes that will break things here. So a safer thing to do (at least from the point of view of robustness) is to check out a commit that we know works. (Normally, you'd check out a specific version or tag, but they don't have any versions or tags in their repo.) 

```
# check out the rxxr2 repo into the rxxr2 directory
git clone https://github.com/ConradIrwin/rxxr2 rxxr2
# check out a known commit
(cd rxxr2 && git checkout 0eea5e9f0e0cde6c39e0fc12614f64edb6189cd5)
```

That checks out the most recent commit in the repository at the time of this writing.

Reply:
Okay, that makes sense.